### ISSUE TITLE
Crash - Crashlog

### ISSUE THREAD

**Issue by igormclven** [TeamRizu/OutFox#134]
Hi guys, 

I was playing a song as usual, but this time the game crashed. This is the log, in case there are any clues as to what might have happened.

```
StepMania5.3-git-a7f5b4623d crash report (build a7f5b4623d, 20201007 @ 01:05:12)
--------------------------------------

Crash reason: Access Violation

Thread: main
        ScreenGameplay.cpp:2727 HandleScreenMessage(SM_None)
        Screen.cpp:223 ScreenMessage(SM_100Combo)
        ScreenGameplay.cpp:2727 HandleScreenMessage(SM_100Combo)
        Screen.cpp:223 ScreenMessage(SM_ComboStopped)
        ScreenGameplay.cpp:2727 HandleScreenMessage(SM_ComboStopped)
Thread: Decode thread
        RageSoundDriver_Generic_Software.cpp:204 Processing the sound while buffers are available.
        RageSoundDriver_Generic_Software.cpp:204 Processing the sound while buffers are available.
        RageSoundDriver_Generic_Software.cpp:204 Processing the sound while buffers are available.
        RageSoundDriver_Generic_Software.cpp:204 Processing the sound while buffers are available.
        RageSoundDriver_Generic_Software.cpp:204 Processing the sound while buffers are available.
Thread: Mixer thread
Thread: Music thread
        RageFileDriverDirectHelpers.cpp:229 F:/Games/StepMania 5.3 Outfox/Appearance/Themes/default/*
        RageFileDriverDirectHelpers.cpp:229 F:/Games/StepMania 5.3 Outfox/Appearance/Themes/default/Sounds/*
        RageFileDriverDirectHelpers.cpp:229 F:/Games/StepMania 5.3 Outfox/Appearance/Themes/_fallback/Sounds/*
        RageFileDriverDirectHelpers.cpp:229 F:/Games/StepMania 5.3 Outfox/Appearance/Themes/_fallback/Sounds/*
        RageFileDriverDirectHelpers.cpp:229 F:/Games/StepMania 5.3 Outfox/Appearance/Themes/_fallback/Sounds/*
Thread: Worker thread (MemoryCardWorker)
Thread: Worker thread (/@mc1int/)
Thread: Worker thread (/@mc2int/)
```

Thank u ❤

**Comment by Jousway** [comment]
thanks for the report! :)
can you get it to crash every time? also I would like the log file in stepmania/logs please to see what was going on :)

**Comment by igormclven** [comment]
It's happening often, it didn't happen before, I don't know what changed. 

This is the full report:
```
StepMania5.3-git-a7f5b4623d crash report (build a7f5b4623d, 20201007 @ 01:05:12)
--------------------------------------

Crash reason: Access Violation

Thread: main
        ScreenGameplay.cpp:2727 HandleScreenMessage(SM_ComboStopped)
        Screen.cpp:223 ScreenMessage(SM_100Combo)
        ScreenGameplay.cpp:2727 HandleScreenMessage(SM_100Combo)
        Screen.cpp:223 ScreenMessage(SM_ComboStopped)
        ScreenGameplay.cpp:2727 HandleScreenMessage(SM_ComboStopped)
Thread: Decode thread
        RageSoundDriver_Generic_Software.cpp:204 Processing the sound while buffers are available.
        RageSoundDriver_Generic_Software.cpp:204 Processing the sound while buffers are available.
        RageSoundDriver_Generic_Software.cpp:204 Processing the sound while buffers are available.
        RageSoundDriver_Generic_Software.cpp:204 Processing the sound while buffers are available.
        RageSoundDriver_Generic_Software.cpp:204 Processing the sound while buffers are available.
Thread: Mixer thread
Thread: Music thread
        RageFileDriverDirectHelpers.cpp:229 F:/Games/StepMania 5.3 Outfox/Appearance/Themes/*
        RageFileDriverDirectHelpers.cpp:229 F:/Games/StepMania 5.3 Outfox/Appearance/Themes/default/Sounds/*
        RageFileDriverDirectHelpers.cpp:229 F:/Games/StepMania 5.3 Outfox/Appearance/Themes/default/*
        RageFileDriverDirectHelpers.cpp:229 F:/Games/StepMania 5.3 Outfox/Appearance/Themes/default/Sounds/*
        RageFileDriverDirectHelpers.cpp:229 F:/Games/StepMania 5.3 Outfox/Appearance/Themes/_fallback/Sounds/*
Thread: Worker thread (MemoryCardWorker)
Thread: Worker thread (/@mc1int/)
Thread: Worker thread (/@mc2int/)
70986005: swscale-5!sws_scale [70950000+35ff0+15]
76e44e69: msvcrt!aligned_offset_malloc [76e00000+44de0+89]
76e44db7: msvcrt!aligned_free [76e00000+44da0+17]
70998860: swscale-5!sws_convertPalette8ToPacked24 [70950000+3fdd0+8a90]
7099b041: swscale-5!sws_getCachedContext [70950000+4ad90+2b1]
6c8c24f0: avutil-56!av_mallocz [6c890000+32490+60]
0063ea3c: StepMania!003f0000
00588367: StepMania!003f0000
0054ed0b: StepMania!003f0000
0055c6e9: StepMania!003f0000
00779282: StepMania!003f0000
0054e9a6: StepMania!003f0000
0077901a: StepMania!003f0000
0059a6f4: StepMania!003f0000
0059a1cb: StepMania!003f0000
0059a1cb: StepMania!003f0000
0055c6e9: StepMania!003f0000
0055c6ff: StepMania!003f0000
0055c857: StepMania!003f0000
004cd5d9: StepMania!003f0000
004cd57f: StepMania!003f0000
0054e9bd: StepMania!003f0000
0059a158: StepMania!003f0000
0054ed0b: StepMania!003f0000
0055c6e9: StepMania!003f0000
00588367: StepMania!003f0000
0054e9a6: StepMania!003f0000
6a9035c7: atioglxx!DrvPresentBuffers [69130000+731f70+10a1657]
0054ed0b: StepMania!003f0000
0055c6e9: StepMania!003f0000
0054e9a6: StepMania!003f0000
6a8c739a: atioglxx!DrvPresentBuffers [69130000+731f70+106542a]
00853464: StepMania!003f0000
6a902875: atioglxx!DrvPresentBuffers [69130000+731f70+10a0905]
69886027: atioglxx!DrvPresentBuffers [69130000+731f70+240b7]
6a902875: atioglxx!DrvPresentBuffers [69130000+731f70+10a0905]
6a4dffaf: atioglxx!DrvPresentBuffers [69130000+731f70+c7e03f]
6a296a2b: atioglxx!DrvPresentBuffers [69130000+731f70+a34abb]
6a8c739a: atioglxx!DrvPresentBuffers [69130000+731f70+106542a]
6a8cdade: atioglxx!DrvPresentBuffers [69130000+731f70+106bb6e]
6a887826: atioglxx!DrvPresentBuffers [69130000+731f70+10258b6]
6a902875: atioglxx!DrvPresentBuffers [69130000+731f70+10a0905]
00aae03f: StepMania!003f0000
6a40ed8b: atioglxx!DrvPresentBuffers [69130000+731f70+bace1b]
6a9035c7: atioglxx!DrvPresentBuffers [69130000+731f70+10a1657]
6a90269b: atioglxx!DrvPresentBuffers [69130000+731f70+10a072b]
6a8cdade: atioglxx!DrvPresentBuffers [69130000+731f70+106bb6e]
6a887826: atioglxx!DrvPresentBuffers [69130000+731f70+10258b6]
6a902875: atioglxx!DrvPresentBuffers [69130000+731f70+10a0905]
6a512f1c: atioglxx!DrvPresentBuffers [69130000+731f70+cb0fac]
69876522: atioglxx!DrvPresentBuffers [69130000+731f70+145b2]
6a90269b: atioglxx!DrvPresentBuffers [69130000+731f70+10a072b]
6a9035c7: atioglxx!DrvPresentBuffers [69130000+731f70+10a1657]
6a1fdef8: atioglxx!DrvPresentBuffers [69130000+731f70+99bf88]
6a7bba02: atioglxx!DrvPresentBuffers [69130000+731f70+f59a92]
6a7bbaf0: atioglxx!DrvPresentBuffers [69130000+731f70+f59b80]
6a7bbe4a: atioglxx!DrvPresentBuffers [69130000+731f70+f59eda]
6a7bbe4a: atioglxx!DrvPresentBuffers [69130000+731f70+f59eda]
6a8df220: atioglxx!DrvPresentBuffers [69130000+731f70+107d2b0]
6a52b876: atioglxx!DrvPresentBuffers [69130000+731f70+cc9906]
7748578c: ntdll!RtlAllocateHeap [77440000+453c0+3cc]
774853fe: ntdll!RtlAllocateHeap [77440000+453c0+3e]
774853fe: ntdll!RtlAllocateHeap [77440000+453c0+3e]
7605a30a: RPCRT4!NdrConformantArrayUnmarshall [76040000+1a070+29a]
7605ef8f: RPCRT4!NdrPointerUnmarshall [76040000+1e710+87f]
7605ee34: RPCRT4!NdrPointerUnmarshall [76040000+1e710+724]
7605c27c: RPCRT4!NdrComplexStructMarshall [76040000+1a830+1a4c]
7748a0d4: ntdll!RtlInterlockedPushEntrySList [77440000+4a0a0+34]
76070b1e: RPCRT4!I_RpcGetBufferWithObject [76040000+2f900+121e]
76046a34: RPCRT4!NdrClientCall2 [76040000+5970+10c4]
760469e8: RPCRT4!NdrClientCall2 [76040000+5970+1078]
76045fbc: RPCRT4!NdrClientCall2 [76040000+5970+64c]
76045fda: RPCRT4!NdrClientCall2 [76040000+5970+66a]
76046a34: RPCRT4!NdrClientCall2 [76040000+5970+10c4]
774771e1: ntdll!SbSelectProcedure [77440000+370d0+111]
77477238: ntdll!SbSelectProcedure [77440000+370d0+168]
769ef7a9: KERNELBASE!WaitForSingleObjectEx [768e0000+10f710+99]
769ef7c6: KERNELBASE!WaitForSingleObjectEx [768e0000+10f710+b6]
769ef646: KERNELBASE!CloseHandle [768e0000+10f620+26]
769ef66a: KERNELBASE!CloseHandle [768e0000+10f620+4a]
712ac275: wdmaud!widMessage [71290000+19460+2e15]
712ac27e: wdmaud!widMessage [71290000+19460+2e1e]
712aa454: wdmaud!widMessage [71290000+19460+ff4]
712aa454: wdmaud!widMessage [71290000+19460+ff4]
712a0cd5: wdmaud!midMessage [71290000+c730+45a5]
7129930c: wdmaud!wodMessage [71290000+9180+18c]
77485695: ntdll!RtlAllocateHeap [77440000+453c0+2d5]
00779282: StepMania!003f0000
0077901a: StepMania!003f0000
006d1db6: StepMania!003f0000
006d1dd7: StepMania!003f0000
004cd5d9: StepMania!003f0000
004cd57f: StepMania!003f0000
004d2ab1: StepMania!003f0000
00817c0e: StepMania!003f0000
008aa533: StepMania!003f0000
00815ab4: StepMania!003f0000
0072eba7: StepMania!003f0000
005101e8: StepMania!003f0000

Static log:
StepMania5.3-git-a7f5b4623d
Compiled 20201007 @ 01:05:12 (build a7f5b4623d)
Log starting 2020-10-25 18:04:58
WARNING: OutFox Engine: Failed to initialise Discord SDK Core! - Exiting
Loading window: sdl
Windows 10
OutFox Engine - Memory Detected: 16318MB Installed, (10784MB Available), 29630MB Swap Memory (20038MB Swap available)
Video driver: Radeon RX 580 Series [Advanced Micro Devices, Inc.]
              27.20.12033.1007, 10-16-2020 [PCI\VEN_1002&DEV_67DF&REV_E7]
Drive: "        ST1000LM024 HN-M2BA3" Driver: storahci DMA: N/A
Drive: "        ST1500DL003-9VT1CC32" Driver: storahci DMA: N/A
Drive: "NVMe    WDS500G3XHC-00SJ00WD" Driver: stornvme DMA: N/A
Drive: "NVMe    Samsung SSD 970 EXM7" Driver: stornvme DMA: N/A
Sound device 0: Headphones (High Definition Aud, 0.0, MID 1, PID 65535 
Sound device 1: 3 - LED MONITOR (AMD High Defin, 10.0, MID 1, PID 100 
Sound device 2: 6 - C24F390 (AMD High Definitio, 10.0, MID 1, PID 100 
WaveOut software mixing at 44100 hz
Sound driver: WaveOut
Lights driver: SystemMessage
Lights driver: Export
Video renderers: 'glad,opengl,gles2'
Graphics Manager:  OpenGL Vendor: ATI Technologies Inc.
Graphics Manager:  OpenGL Renderer: Radeon RX 580 Series
Graphics Manager:  OpenGL Version: 4.5.14736 Compatibility Profile Context 20.10.1 27.20.12033.1007
Graphics Manager:  OpenGL Max texture size: 16384
Graphics Manager:  OpenGL Texture units: 8
Graphics Manager:  OpenGL Extensions:
  GL_AMDX_debug_output
  GL_AMD: blend_minmax_factor, conservative_depth, debug_output, depth_clamp_separate, draw_buffers_blend, 
    framebuffer_sample_positions, gcn_shader, gpu_shader_half_float, gpu_shader_int16, gpu_shader_int64, 
    interleaved_elements, multi_draw_indirect, name_gen_delete, occlusion_query_event, performance_monitor, 
    pinned_memory, query_buffer_object, sample_positions, seamless_cubemap_per_texture, shader_atomic_counter_ops, 
    shader_stencil_export, shader_stencil_value_export, shader_trace, shader_trinary_minmax, sparse_texture, 
    sparse_texture_pool, stencil_operation_extended, texture_cube_map_array, texture_texture4, 
    transform_feedback3_lines_triangles, transform_feedback4, vertex_shader_layer, vertex_shader_viewport_index
  GL_ARB: ES2_compatibility, ES3_1_compatibility, ES3_compatibility, arrays_of_arrays, base_instance, 
    bindless_texture, blend_func_extended, buffer_storage, clear_buffer_object, clear_texture, clip_control, 
    color_buffer_float, compatibility, compressed_texture_pixel_storage, compute_shader, 
    conditional_render_inverted, conservative_depth, copy_buffer, copy_image, cull_distance, debug_output, 
    depth_buffer_float, depth_clamp, depth_texture, derivative_control, direct_state_access, draw_buffers, 
    draw_buffers_blend, draw_elements_base_vertex, draw_indirect, draw_instanced, enhanced_layouts, 
    explicit_attrib_location, explicit_uniform_location, fragment_coord_conventions, fragment_layer_viewport, 
    fragment_program, fragment_program_shadow, fragment_shader, framebuffer_no_attachments, framebuffer_object, 
    framebuffer_sRGB, geometry_shader4, get_program_binary, get_texture_sub_image, gl_spirv, gpu_shader5, 
    gpu_shader_fp64, half_float_pixel, half_float_vertex, imaging, indirect_parameters, instanced_arrays, 
    internalformat_query, internalformat_query2, invalidate_subdata, map_buffer_alignment, map_buffer_range, 
    multi_bind, multi_draw_indirect, multisample, multitexture, occlusion_query, occlusion_query2, 
    parallel_shader_compile, pipeline_statistics_query, pixel_buffer_object, point_parameters, point_sprite, 
    polygon_offset_clamp, program_interface_query, provoking_vertex, query_buffer_object, 
    robust_buffer_access_behavior, sample_shading, sampler_objects, seamless_cube_map, 
    seamless_cubemap_per_texture, separate_shader_objects, shader_atomic_counter_ops, shader_atomic_counters, 
    shader_ballot, shader_bit_encoding, shader_draw_parameters, shader_group_vote, shader_image_load_store, 
    shader_image_size, shader_objects, shader_precision, shader_stencil_export, shader_storage_buffer_object, 
    shader_subroutine, shader_texture_image_samples, shader_texture_lod, shader_viewport_layer_array, 
    shading_language_100, shading_language_420pack, shading_language_packing, shadow, shadow_ambient, 
    sparse_buffer, sparse_texture, spirv_extensions, stencil_texturing, sync, tessellation_shader, 
    texture_barrier, texture_border_clamp, texture_buffer_object, texture_buffer_object_rgb32, 
    texture_buffer_range, texture_compression, texture_compression_bptc, texture_compression_rgtc, 
    texture_cube_map, texture_cube_map_array, texture_env_add, texture_env_combine, texture_env_crossbar, 
    texture_env_dot3, texture_float, texture_gather, texture_mirror_clamp_to_edge, texture_mirrored_repeat, 
    texture_multisample, texture_non_power_of_two, texture_query_levels, texture_query_lod, texture_rectangle, 
    texture_rg, texture_rgb10_a2ui, texture_snorm, texture_stencil8, texture_storage, texture_storage_multisample, 
    texture_swizzle, texture_view, timer_query, transform_feedback2, transform_feedback3, 
    transform_feedback_instanced, transform_feedback_overflow_query, transpose_matrix, uniform_buffer_object, 
    vertex_array_bgra, vertex_array_object, vertex_attrib_64bit, vertex_attrib_binding, vertex_buffer_object, 
    vertex_program, vertex_shader, vertex_type_10f_11f_11f_rev, vertex_type_2_10_10_10_rev, viewport_array, 
    window_pos
  GL_ATI: draw_buffers, envmap_bumpmap, fragment_shader, separate_stencil, texture_compression_3dc, 
    texture_env_combine3, texture_float, texture_mirror_once
  GL_EXT: abgr, bgra, bindable_uniform, blend_color, blend_equation_separate, blend_func_separate, blend_minmax, 
    blend_subtract, compiled_vertex_array, copy_buffer, copy_texture, depth_bounds_test, direct_state_access, 
    draw_buffers2, draw_instanced, draw_range_elements, fog_coord, framebuffer_blit, framebuffer_multisample, 
    framebuffer_object, framebuffer_sRGB, geometry_shader4, gpu_program_parameters, gpu_shader4, histogram, 
    memory_object, memory_object_win32, multi_draw_arrays, packed_depth_stencil, packed_float, packed_pixels, 
    pixel_buffer_object, point_parameters, polygon_offset_clamp, provoking_vertex, rescale_normal, 
    secondary_color, semaphore, semaphore_win32, separate_specular_color, shader_image_load_store, 
    shader_integer_mix, shadow_funcs, stencil_wrap, subtexture, texgen_reflection, texture3D, texture_array, 
    texture_buffer_object, texture_compression_bptc, texture_compression_latc, texture_compression_rgtc, 
    texture_compression_s3tc, texture_cube_map, texture_edge_clamp, texture_env_add, texture_env_combine, 
    texture_env_dot3, texture_filter_anisotropic, texture_integer, texture_lod, texture_lod_bias, 
    texture_mirror_clamp, texture_object, texture_rectangle, texture_sRGB, texture_sRGB_R8, texture_sRGB_RG8, 
    texture_sRGB_decode, texture_shared_exponent, texture_snorm, texture_storage, texture_swizzle, timer_query, 
    transform_feedback, vertex_array, vertex_array_bgra, vertex_attrib_64bit
  GL_IBM_texture_mirrored_repeat
  GL_KHR: context_flush_control, debug, no_error, parallel_shader_compile, robust_buffer_access_behavior, 
    robustness
  GL_KTX_buffer_region
  GL_NV: alpha_to_coverage_dither_control, blend_square, conditional_render, copy_depth_to_color, copy_image, 
    depth_buffer_float, explicit_multisample, float_buffer, half_float, primitive_restart, shader_atomic_int64, 
    texgen_reflection, texture_barrier
  GL_OES_EGL_image
  GL_SGIS: generate_mipmap, texture_edge_clamp, texture_lod
  GL_SUN_multi_draw_arrays
  GL_WIN_swap_hint
  WGL_EXT_swap_control
GLAD Windowed 1280x720 32 color 32 texture 60Hz Vsync SmoothLines
WARNING: RageSoundDriver::GetHardwareFrame: too many tries
WARNING: Image cache for '/Songs/9Volt Battery Pack/Altar/altar-bn.png' wasn't loaded
WARNING: Image cache for '/Songs/8 - Cirque du Zeppelin/10 Minutes - [Zeph + Zaia]/../Zeppelin_bn.png' wasn't loaded
WARNING: Image cache for '/Songs/8 - Cirque du Zeppelin/A Brighter Day - [Zeph + Zaia]/../Zeppelin_bn.png' wasn't loaded
WARNING: Image cache for '/Songs/8 - Cirque du Zeppelin/Amazing - [Zaia]/../Zeppelin_bn.png' wasn't loaded
Players joined: P1
Top Screen: ScreenGameplay
Language: en
Current renderer: OpenGL
Theme: default

Partial log:
13:48.577: 60 FPS, 59 av FPS, 1610 VPF, 1368 MB, GLAD
13:49.578: 60 FPS, 59 av FPS, 1595 VPF, 1368 MB, GLAD
13:50.578: 60 FPS, 59 av FPS, 1624 VPF, 1368 MB, GLAD
13:51.580: 60 FPS, 59 av FPS, 1582 VPF, 1368 MB, GLAD
13:52.579: 60 FPS, 59 av FPS, 1589 VPF, 1368 MB, GLAD
13:53.582: 60 FPS, 59 av FPS, 1212 VPF, 1368 MB, GLAD
13:54.582: 60 FPS, 59 av FPS, 539 VPF, 1368 MB, GLAD
13:55.583: 60 FPS, 59 av FPS, 1396 VPF, 1368 MB, GLAD
13:56.585: 60 FPS, 59 av FPS, 1631 VPF, 1368 MB, GLAD
13:57.585: 60 FPS, 59 av FPS, 1605 VPF, 1368 MB, GLAD

-- End of report
```

**Comment by Scraticus** [comment]
can you send us the log.txt as well? many thanks

**Comment by Scraticus** [comment]
Fixed in 4.9.7GG
